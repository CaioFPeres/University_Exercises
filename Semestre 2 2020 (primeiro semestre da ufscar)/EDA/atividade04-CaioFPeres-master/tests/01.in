9
8 3 10 14 6 4 13 7 1
