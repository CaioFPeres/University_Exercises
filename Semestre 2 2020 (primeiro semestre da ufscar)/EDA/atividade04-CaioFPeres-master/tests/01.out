Pr.: 8 3 1 6 4 7 10 14 13
In.: 1 3 4 6 7 8 10 13 14
Po.: 1 4 7 6 3 13 14 10 8
Ni.: 8 3 10 1 6 14 4 7 13
